src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageSourceMappingActionControl.xaml

```diff
@@ -18,12 +18,16 @@
     </ResourceDictionary>
   </UserControl.Resources>
   <Grid>
-    <StackPanel Orientation="Horizontal">
+    <Grid.ColumnDefinitions>
+      <ColumnDefinition Width="auto"/>
+      <ColumnDefinition/>
+    </Grid.ColumnDefinitions>
       <imaging:CrispImage
+        Grid.Column="0"
```

Comment:
Why was this converted to a Grid? Can you please add a screenshot showing the problem it solved (before/after)?